tracer/src/Datadog.Trace/Processors/Obfuscator.cs

```diff
@@ -0,0 +1,175 @@
+// <copyright file="Obfuscator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.TraceProcessors
+{
+    // https://github.com/DataDog/dd-trace-java/blob/35487fa08f16503105b2ff37fb084ffa5c894f24/internal-api/src/main/java/datadog/trace/api/normalize/SQLNormalizer.java
+
+    internal class Obfuscator
+    {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<Obfuscator>();
+        private static readonly BitArray NumericLiteralPrefix = new BitArray(256, false);
+        private static readonly BitArray Splitters = new BitArray(256, false);
+
+        static Obfuscator()
```

Comment:
nit - unless you want to use it somewhere else, we could have added this code into the `ObfuscatorTraceProcessor ` class

Reply:
Not sure how I would about adding that part of the code to that class, do you also mean the  `static Obfuscator()`
method? If so I'm curious to know the pros of doing so.

Reply:
No I meant move the whole content into the other class. Cause `ObfuscatorTraceProcessor` is just calling this class, nothing else. 
But it's a nit. Nit stands for nitpicking, so in my vocabulary, it means completely optional.

Reply:
Makes sense, I agree and moved the logic to `ObfuscatorTraceProcessor` instead.